documentation/getting_started.md

```diff
@@ -44,8 +44,8 @@ The previous example will generate the following policies:
 
 ```elixir
 field_policies do
-  field_policy [:name, :email]do
-    authorize_if {AshRbac.HasRole, [role: [:user]]}
+  field_policy [:name, :email], [{AshRbac.HasRole, [role: [:user]]}] do
+    authorize_if always()
```

Comment:
The generated policy changed a bit. I moved the role-check up into the condition. I feel like this way, it is easier to also add other field_policies in the field_policies block. Because the generated rules only apply if the user is allowed to see a field. With the old style, it was sometimes more complicated because the policy block would apply but not authorize because the role didn't match, blocking access even if a custom policy would allow it in the field_policy block.